packages/flutter_tools/test/commands/run_test.dart

```diff
@@ -28,6 +43,120 @@ void main() {
       }
     });
 
+
+    group('dart-flags option', () {
+      testUsingContext('is not available on stable channel', () async {
+        when(mockDeviceManager.getDevices()).thenAnswer((Invocation invocation) {
```

Comment:
It looks like you can remove the repeated parts (49 - 60) here to a setUp() under the group.

Reply:
Done.